include/dali/core/error_handling.h

```diff
@@ -233,29 +247,32 @@ inline dali::string GetStacktrace() {
 #define DALI_STR(x) DALI_STR2(x)
 #define FILE_AND_LINE __FILE__ ":" DALI_STR(__LINE__)
 
-#define DALI_MESSAGE_WITH_STACKTRACE(str)\
-  (std::string("[" FILE_AND_LINE "] ") + str + dali::GetStacktrace())
+#define DALI_MESSAGE_WITH_STACKTRACE(...)\
+  (std::string("[" FILE_AND_LINE "] ")), ##__VA_ARGS__, dali::GetStacktrace()
```

Comment:
```suggestion
#define DALI_MESSAGE_AND_STACKTRACE(...)\
  (std::string("[" FILE_AND_LINE "] ")), ##__VA_ARGS__, dali::GetStacktrace()
```
Now it's no longer a message with stack trace - those are two separate things.

Reply:
done